packages/cosmos_ui_components/lib/components/cosmos_text_field.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:flutter/material.dart';
+
+class CosmosTextField extends StatefulWidget {
+  final String text;
+  final Function(String) onChanged;
+  final int? maxLines;
+  final int maxLength;
+
+  const CosmosTextField({
+    Key? key,
+    required this.onChanged,
+    this.maxLength = 50,
+    this.text = '',
+    this.maxLines,
+  }) : super(key: key);
+
+  @override
+  State<CosmosTextField> createState() => _CosmosTextFieldState();
+}
+
+class _CosmosTextFieldState extends State<CosmosTextField> {
+  final TextEditingController controller = TextEditingController();
```

Comment:
you should initialize controller in `initState` and dispose it in `dispose`

Reply:
Updated